src/miner.cpp

```diff
@@ -855,6 +1041,73 @@ void StakeMiner(CWallet *pwallet)
     RenameThread("grc-stake-miner");
 
     MinerAutoUnlockFeature(pwallet);
+    
+    // Parse StakeSplit and SideStaking flags.
+    bool fEnableStakeSplit = GetBoolArg("-enablestakesplit");
+    LogPrintf("StakeMiner: fEnableStakeSplit = %u", fEnableStakeSplit);
+
+    bool fEnableSideStaking = GetBoolArg("-enablesidestaking");
+    LogPrintf("StakeMiner: fEnableSideStaking = %u", fEnableSideStaking);
+
+    vector<string> vSubParam;
+    SideStakeAlloc vSideStakeAlloc;
+    std::string sAddress;
+    int64_t nMinStakeSplitValue;
+    double dEfficiency;
+    double dAllocation;
+    
+    // If side staking is enabled, parse destinations and allocations.
+    if (fEnableSideStaking)
+    {
+        if (mapArgs.count("-sidestake") && mapMultiArgs["-sidestake"].size() > 0)
```

Comment:
if SideStaking is enabled but no -sidestake is set at all in config, we should log a warning message to state that side staking is enabled however no sidestake parameters have been set. lets be vocal where we can to assist a user who may of forgotten or not set the config options to support sidestaking.

Reply:
I agree!

Reply:
I added a comment at 1061 where the parsing is to ensure people know that skipped allocations result in the rewards going back into the coinstake... and I added a check for dSumAllocation == 0 at 727 and if it is, prints a warning message in debug.log.